libs/sdk-backend-bear/src/convertors/fromBackend/ObjRefConverter.ts

```diff
@@ -10,12 +10,12 @@ import { isLocalIdRef, isUriRef, ObjectType, ObjRef, ObjRefInScope } from "@good
  * @param defaultType - type to use it the ref has none specified
  * @internal
  */
-export function fromBearRef(ref: ObjRef, defaultType?: ObjectType): ObjRef {
-    if (isUriRef(ref)) {
-        return ref;
+export function fromBearRef(ref: GdcVisualizationObject.ObjQualifier, defaultType?: ObjectType): ObjRef {
+    if (GdcExecuteAFM.isObjectUriQualifier(ref)) {
+        return uriRef(ref.uri);
     }
 
-    return { identifier: ref.identifier, type: ref.type ?? defaultType };
+    return idRef(ref.identifier, defaultType);
```

Comment:
Its ok this conversion when defaultType is missing and ref.type present ?

Reply:
Basically we can do this params as required, because all places have this apram filled